include/simdutf/implementation.h

```diff
@@ -6,6 +6,9 @@
 #endif
 #include <vector>
 #include <tuple>
+#if __cplusplus >= 202002L
```

Comment:
Could be handy to have a macro like... `SIMDUTF_SPAN`.

Note that Visual Studio does not necessarily abide by __cplusplus...